common/djangoapps/util/views.py

```diff
@@ -318,18 +316,6 @@ def build_error_response(status_code, field, err_msg):
         [(tag, request.POST[tag]) for tag in ["issue_type", "course_id"] if tag in request.POST]
     )
 
-    if UserProfile.has_registered(request.user):
```

Comment:
I think we want this logic, just maybe need to change the `has_registered` call.